### ISSUE TITLE
Some missed injections in .sbncs

### ISSUE THREAD

**Issue by mortenn** [lunet-io/scriban-vscode#9]
When using `{{ }}` inside a C# string or to inject a type into a lambda argument list, the syntax highlighting appears to fail:

![image](https://user-images.githubusercontent.com/640095/135276307-ed6f7e09-cf1e-4d56-be7c-9b76c0651c9b.png)

**Comment by xoofx** [comment]
Agreed, it would require to rewrite the string rule (and wouldn't be perfect likely)
